panther_gazebo/launch/simulation.launch.py

```diff
@@ -71,6 +72,17 @@ def generate_launch_description():
         ),
     )
 
+    use_docking = LaunchConfiguration("use_docking")
```

Comment:
Why there is no declaration?

Reply:
ROS Launch is not a miracle package when it comes to arguments.
On the one hand, the arguments are inherited, so they only need to be in the lowest level launch. What is very convenient, e.g. rosbot and panther can include launch from husarion_gz_worlds and you do not need to add arguments to the map.
On the other hand, overriding an argument requires declaring the argument again (for the `-s` flag to work properly) and overwriting it in the launch_arguments field (for the change to be visible).

Reply:
So, should I add the declaration?